automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookExtensionSpec.scala

```diff
@@ -0,0 +1,116 @@
+package org.broadinstitute.dsde.workbench.leonardo.notebooks
+
+import akka.http.scaladsl.model.HttpResponse
+import org.broadinstitute.dsde.workbench.ResourceFile
+import org.broadinstitute.dsde.workbench.google2.GcsBlobName
+import org.broadinstitute.dsde.workbench.leonardo._
+import org.broadinstitute.dsde.workbench.leonardo.notebooks.Notebook.NotebookMode
+import org.scalatest.time.{Minutes, Seconds, Span}
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+
+class NotebookExtensionSpec extends ClusterFixtureSpec with NotebookTestUtils {
+  override def enableWelder: Boolean = true
+
+  "Leonardo welder and jupyter extensions" - {
+
+    "Welder should be up" in { clusterFixture =>
+      val resp: HttpResponse = Welder.getWelderStatus(clusterFixture.cluster)
+      resp.status.isSuccess() shouldBe true
+    }
+
+    "open notebook in edit mode should work" in { clusterFixture =>
+      val sampleNotebook = ResourceFile("bucket-tests/gcsFile.ipynb")
+      val isEditMode = true
+      withResourceFileInBucket(clusterFixture.billingProject, sampleNotebook, "text/plain") { googleCloudDir =>
+        withWelderInitialized(clusterFixture.cluster, googleCloudDir, isEditMode) { localizedFile =>
```

Comment:
I would rename `googleCloudDir` to `gcsPath` since it's a gcs path to an object, not just a directory